iothub/device/src/Transport/FileUploadNotificationResponse.cs

```diff
@@ -1,22 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information
-
-using Newtonsoft.Json;
-
-namespace Microsoft.Azure.Devices.Client.Transport
-{
-    internal class FileUploadNotificationResponse
```

Comment:
Were these unreferenced classes?

Reply:
These are the same Json objects that were used for this flow originally. They were just internal, and I changed their names to fit what they each did

Reply:
Even though github didn't interpret it this way, this class was just renamed to FileUploadSasUriResponse